activitypub.php

```diff
@@ -32,6 +32,7 @@
 \defined( 'ACTIVITYPUB_MAX_IMAGE_ATTACHMENTS' ) || \define( 'ACTIVITYPUB_MAX_IMAGE_ATTACHMENTS', 3 );
 \defined( 'ACTIVITYPUB_HASHTAGS_REGEXP' ) || \define( 'ACTIVITYPUB_HASHTAGS_REGEXP', '(?:(?<=\s)|(?<=<p>)|(?<=<br>)|^)#([A-Za-z0-9_]+)(?:(?=\s|[[:punct:]]|$))' );
 \defined( 'ACTIVITYPUB_USERNAME_REGEXP' ) || \define( 'ACTIVITYPUB_USERNAME_REGEXP', '(?:([A-Za-z0-9\._-]+)@((?:[A-Za-z0-9_-]+\.)+[A-Za-z]+))' );
+\defined( 'ACTIVITYPUB_URLS_REGEXP' ) || \define( 'ACTIVITYPUB_URLS_REGEXP', '(http|https):\/\/([-a-zA-Z0-9@:;\/%_;\+.~#?&=]*)' );
```

Comment:
```suggestion
\defined( 'ACTIVITYPUB_URLS_REGEXP' ) || \define( 'ACTIVITYPUB_URLS_REGEXP', '(www|http:|https:)+[^\s]+[\w]' );
```

@danielhuesken what do you think of this REGEX instead? This would allow umlauts in GET params and the domain!?!

Reply:
@pfefferle looks good for me because of not change urls in html tags it works now.